examples/external_match.py

```diff
@@ -0,0 +1,108 @@
+import os
+import asyncio
+from dotenv import load_dotenv
+from web3 import AsyncWeb3, Web3
+from eth_account import Account
+from eth_account.signers.local import LocalAccount
+from web3.middleware import SignAndSendRawMiddlewareBuilder
+from renegade import ExternalMatchClient
+from renegade.types import AtomicMatchApiBundle, OrderSide, ExternalOrder
+
+# Constants
+BASE_MINT = "0xc3414a7ef14aaaa9c4522dfc00a4e66e74e9c25a"  # Testnet wETH
+QUOTE_MINT = "0xdf8d259c04020562717557f2b5a3cf28e92707d1"  # Testnet USDC
+
+async def get_wallet() -> tuple[AsyncWeb3, LocalAccount]:
+    rpc_url = os.getenv("RPC_URL")
+    if not rpc_url:
+        raise ValueError("RPC_URL environment variable not set")
+
+    w3 = AsyncWeb3(AsyncWeb3.AsyncHTTPProvider(rpc_url))
+    chain_id = await w3.eth.chain_id
```

Comment:
nit: this variable is unused, meaning this whole method does not need to be async